packages/browser-extension/src/scripts/background.js

```diff
@@ -67,7 +69,8 @@ const report = async (message, tab, callback) => {
     const reason = message.reason;
     const userAgent = message.userAgent;
     const version = browser.runtime.getManifest().version;
-    const body = JSON.stringify({ reason, url: tab.url, userAgent, version });
+    const explanation = message.explanation;
```

Comment:
Please, put this line in top of `const reason = ...`.

Reply:
done